apiserver/facades/agent/uniter/uniter.go

```diff
@@ -2411,3 +2411,38 @@ func (u *UniterAPI) SetContainerSpec(args params.SetContainerSpecParams) (params
 	}
 	return results, nil
 }
+
+// GoalStates returns the list with charm peers and related units information.
```

Comment:
Wording a little wonky. Maybe
GoalStates returns the status of the the charm peers and related units.